core/templates/pages/blog-dashboard-page/modal-templates/blog-card-preview-modal.component.ts

```diff
@@ -70,5 +70,10 @@ export class BlogCardPreviewModalComponent
       this.blogPostData.lastUpdated,
       dateString,
     );
+    this.blogHomePageLink = (
```

Comment:
Not sure I understand this, could you please explain why this needs to be in the component TS file 
(i.e. rather than using it directly in the HTML)?